packages/schemas/src/foundations/jsonb-types/file-upload.ts

```diff
@@ -0,0 +1,12 @@
+import { number, object, string, type z } from 'zod';
+
+export const uploadFileGuard = object({
+  filepath: string(),
+  mimetype: string(),
```

Comment:
mimetype is not needed since it can be calculated dynamically?

Reply:
Then how can we control what is contained in the zip file? What if they upload `.sh` scripts and other malicious file types in the zip?
I thought we can unzip the files and check the mimetype one by one, if any of them is not allowed, then we can reject the upload throw error.

Reply:
removed